src/utilities/filetypes/EpwFile.cpp

```diff
@@ -3967,6 +3974,31 @@ bool EpwFile::parse(std::istream& ifs, bool storeData) {
     }
   }
 
+  for (unsigned int i = 0; i < epw_strings.size(); i++) {
```

Comment:
wouldn't 

```
for (const auto &epw_string: epw_strings) {
...
...
}
```

be a bit more modern? I don't believe you need the index, unless I missed it.